### ISSUE TITLE
ruff_n_tumble [cr_pdx] Disk 1 won't load

### ISSUE THREAD

**Issue by bnice** [dirkwhoffmann/vAmiga#417]
Hi,
There is an issue with loading  ruff_n_tumble_(1994)(renegade)(disk_1_of_2)[cr_pdx]-amiga ADF . Here is the url where you can get it: 
http://archive.org/download/Commodore_Amiga_TOSEC_2012_04_10/Commodore_Amiga_TOSEC_2012_04_10.zip/Commodore%20Amiga%20%5BTOSEC%20v2012-04-10%5D%2FGames%2F%5BADF%5D%2FCommodore%20Amiga%20-%20Games%20-%20%5BADF%5D%20%28TOSEC%202012-02-27%29%2FRuff%20%27n%27%20Tumble%20%281994%29%28Renegade%29%28Disk%201%20of%202%29%5Bcr%20PDX%5D.zip
 
Ben

**Comment by dirkwhoffmann** [comment]
Nice catch! Something is going wrong that eventually halts the CPU. 

<img width="974" alt="Bildschirmfoto 2020-10-09 um 22 09 34" src="https://user-images.githubusercontent.com/12561945/95628165-cbb8ae80-0a7d-11eb-94a9-7643ceba082e.png">

I did some quick tests with UAE and it seems to be a drive issue. The game refuses to load in UAE if a turbo drive is chosen. All other drives (original, 200%, ..., 800%) work fine. Blitter doesn't seem to play a role here.

**Comment by dirkwhoffmann** [comment]
This game utilizes a custom loader that makes use of the DSKSYNC register. When booting in FSUAE, the following message appears in the debug log: 

> warning: Disk read DMA aborted, 0 words left PC=503ec

This comes from here: 

```
if (!(v & 0x8000)) {
		if (dskdmaen != DSKDMA_OFF) {
			/* Megalomania and Knightmare does this */
			if (dskdmaen == DSKDMA_READ)
				printf("warning: Disk read DMA aborted, %d words left PC=%x\n", dsklength, M68K_GETPC);
			if (dskdmaen == DSKDMA_WRITE) {
				printf("warning: Disk write DMA aborted, %d words left PC=%x\n", dsklength, M68K_GETPC);
...
		dskdmaen = DSKDMA_OFF;
}
```

The game sets the DMAEN bit in DSKLEN while the disk controller is waiting for a SYNC word. As far as I can tell from vAmiga's debug log, my emulator doesn't switch DMA off in this case.

See #421 for a similar bug repot for "Knightmare".

Unexpectedly, Megalomania seems to work well in vAmiga (at least the version I was trying).

**Comment by dirkwhoffmann** [comment]
Fixed in v0.9.13.2

<img width="974" alt="Bildschirmfoto 2020-10-11 um 17 05 58" src="https://user-images.githubusercontent.com/12561945/95682404-5536c080-0be5-11eb-81e5-9b1033e10c4c.png">
